elasticai/creator/nn/quantized_grads/fixed_point/quantize_to_fixed_point.py

```diff
@@ -1,94 +1,75 @@
 import torch
+from torch import Tensor
 
-from .two_complement_fixed_point_config import FixedPointConfigV2
 
-
-def _clamp_(number: torch.Tensor, fxp_conf: FixedPointConfigV2) -> None:
+class Round(torch.autograd.Function):
     """
-    Inplace clamp values to twos complement.
+    This function implements an STE for rounding.
```

Comment:
To provide another example for a short description, i suggest to reword this to something like
  
  `"Round deterministically to nearest neighbour with STE."`